### ISSUE TITLE
Graphics on front page hard to read

### ISSUE THREAD

**Issue by Sfshaza** [flutter/flutter#8604]
I like the new front page, but have noticed that two of the graphics are hard to read and fail contrast tests for accessibility.

<img width="323" alt="screen shot 2017-03-07 at 12 25 59 am" src="https://cloud.githubusercontent.com/assets/6293442/23648111/129c333c-02ce-11e7-96ca-447c528b578f.png">

For example, the "Rendering" box for this graphic has a background of #3D84E1 and a foreground of #757575. I ran it through the [Web Aim](http://webaim.org/resources/contrastchecker/) color checker to this result:

<img width="777" alt="screen shot 2017-03-07 at 12 33 45 am" src="https://cloud.githubusercontent.com/assets/6293442/23648158/4a77c4ba-02ce-11e7-9cfa-df114c79f6a1.png">

----------------------

I also had trouble reading this diagram:

<img width="454" alt="screen shot 2017-03-07 at 12 26 08 am" src="https://cloud.githubusercontent.com/assets/6293442/23648184/64a574a4-02ce-11e7-9128-ad5a37845de2.png">

The background has a color of #6563C8 and a foreground of #0B0B15. The analyzer gave this result:

<img width="792" alt="screen shot 2017-03-07 at 12 34 15 am" src="https://cloud.githubusercontent.com/assets/6293442/23648220/8e86d326-02ce-11e7-8626-5feaec660af6.png">

Also, the font for this diagram is rather spindly. We've gotten complaints on dartlang about the difficulty of reading spindly fonts.

-----

The [Material Design color palette](https://material.io/guidelines/style/color.html#color-color-palette) uses black text on light colors and white text on dark colors. Perhaps something like that, along with a bolder font, could be used here?

**Comment by sethladd** [comment]
See also https://github.com/flutter/flutter/issues/8546

**Comment by Sfshaza** [comment]
I was just talking to David Yang and he mentioned the front page. He suggested that we insert the Shrine screenshots into a simulated phone case. It's too easy to miss that the screenshots show iOS and Android. (In fact, I did miss that.)

I remember that we have, somewhere in our Android docs, instructions for doing this on Android. I searched and couldn't find it, but I found this, which allows you to do it on various platforms:

http://mockuphone.com/

**Comment by Sfshaza** [comment]
Opened https://github.com/flutter/flutter/issues/8625

**Comment by ahmetaa** [comment]
Here is a slightly better version of that awful hierarchy graphic:
Feel free to use it if you like.

![flutter hierarchy](https://cloud.githubusercontent.com/assets/2437047/23692854/eb55dadc-03d9-11e7-8eea-f38f5288176a.png)

Source: 
https://docs.google.com/drawings/d/13jhVeYSf7YwH-a5DmPDYQbB6qvDKSZO5lggRr0xw-vU/edit?usp=sharing

**Comment by ahmetaa** [comment]
Here is a slightly better version of that awful hierarchy graphic:
Feel free to use it if you like.

![flutter hierarchy](https://cloud.githubusercontent.com/assets/2437047/23692952/60e27896-03da-11e7-9482-35c0222efd69.png)

Source: https://docs.google.com/drawings/d/13jhVeYSf7YwH-a5DmPDYQbB6qvDKSZO5lggRr0xw-vU/edit?usp=sharing

**Comment by Sfshaza** [comment]
Definitely more readable, @ahmetaa!

Can I make another suggestion? Store the source for all diagrams in _diagrams in the repo. I have done this for the diagrams in my tutorials. It's much easier to then change them, if necessary.

**Comment by sethladd** [comment]
Can we use SVG on the homepage? Seems like a good fit for something like this.

**Comment by eseidelGoogle** [comment]
SVG is part of HTML these days so presumably.  Our logo used to be svg (not sure what it is now).  apple.com and other big sites use SVG on their homepages. :)

**Comment by Hixie** [comment]
We already use SVG on https://flutter.io/widgets/, about half the pictures are SVG.

**Comment by Sfshaza** [comment]
SVG works for me! But for diagrams created in Omnigraffle or one of the Adobe projects, it would be nice to save those on the site.

**Comment by LarkAscending** [comment]
Wow, lots of opinions. Glad people care so intensely. If you look at the original PR, it says the graphics are provisional and subject to change, though. That's why we didn't sink hours and hours into them. They're just for First Look, for communication purposes.

https://github.com/flutter/website/pull/407

@Sfshaza looks like your review of that PR never happened?

I'm on multiple communuication channels - just ping me if you have free time and get distracted by something like this.


@Sfshaza there are no source files - they're not diagrams, they're PNG files.

**Comment by sethladd** [comment]
Update:  @bkobash has proposed new graphics.

**Comment by sethladd** [comment]
New homepage graphics updated.

**Comment by devxpy** [comment]
I feel like the use of thin fonts on the docs is a bad choice. It really unreadable. The font weight should be kept normal.

**Comment by Hixie** [comment]
Please file a new bug to let us know how we should change the web site! thanks!

**Comment by devxpy** [comment]
#19394

**Comment by Sfshaza** [comment]
@devxpy, we are working on an updated landing page and we should definitely check it against accessibility requirements. @maguinis, @filiph, just a reminder.

At any rate, this bug was originally filed against a very old, previous landing page, not the current page, which was updated for Flutter beta.

**Comment by Sfshaza** [comment]
@maguinis, @InMatrix, just a reminder that we need to subject the site to accessibility testing.
